extensions/resource-deployment/src/ui/wizardBase.ts

```diff
@@ -12,7 +12,7 @@ import { WizardPageBase } from './wizardPageBase';
 import { WizardPageInfo } from './wizardPageInfo';
 const localize = nls.loadMessageBundle();
 
-export abstract class WizardBase<T, P extends WizardPageBase<T>, M extends Model> {
+export abstract class WizardBase<P extends WizardPageBase<WizardBase<P, M>>, M extends Model> {
```

Comment:
1. Am I missing something or is this never even used anywhere?
2. This seems extremely convoluted - it's basically an infinite loop of generics...do you really need to have both the wizard and the pages each know about the specific types of the others?
3. What is this even fixing? I don't understand the description you left - what tests are you referring to?

Reply:
1. It is used.
2. That is the way the generics specifications that depend on each other work apparently. Luckily, there is no infinite loop of generics with this specification.
3. Added details in the description. I previously thought the title of the PR was a sufficient description.

Reply:
Where is it used? I don't see it referenced anywhere 

![image](https://user-images.githubusercontent.com/28519865/98843664-20927080-2400-11eb-914a-35d36d3751e6.png)

Reply:
same here, not sure why is this no longer being used and are you using this base class in your uncommitted code?

Reply:
Ok will remove. I mistakenly thought that rezourcetype wizard stl derives from it.